debian/changelog

```diff
@@ -0,0 +1,5 @@
+python-ovh (0.0.1-1) trusty; urgency=low
```

Comment:
Can you put the actual version number here and include a call to ``dch`` in https://github.com/ovh/python-ovh/blob/master/scripts/bump-version.sh to add debian version bumping as part of the version release ? Otherwise I fear the file will fall behind. Feel free to jab me Monday if you want an example of how to automate such a call.